src/api/app/views/webui/status_messages/_breadcrumb_items.html.haml

```diff
@@ -1,2 +1,4 @@
+= render partial: 'webui/configuration/breadcrumb_items'
+
 %li.breadcrumb-item.active{ 'aria-current' => 'page' }
-  Status Message
+  Manage News
```

Comment:
We really need to be consistent in the naming of news / status messages. I suggest going with news as I wrote in #13574.